experiments/experiment.py

```diff
@@ -145,3 +147,41 @@ def run(self):
 
             # TODO: Stream back stats
             # For now: write to file
+            self.write(self.generate_name(), state)
+
+    def format_data(self, data):
+        # state = {
+        #     "cat": cat,
+        #     "dai": dai,
+        #     "dai_join": dai_join,
+        #     "flapper": flapper,
+        #     "flippers": flippers,
+        #     "flopper": flopper,
+        #     "gem_joins": gem_joins,
+        #     "ilks": ilks,
+        #     "keepers": keepers,
+        #     "spotter": spotter,
+        #     "stats": {},
+        #     "vat": vat,
+        #     "vow": vow,
+        # }
+        for key, value in data.items():
```

Comment:
make sure to make a copy of `data`

Reply:
also req:
let's add a boolean param for whether or not to record full state or stats
in either case, when coming across a `Rad`/`Wad`/`Ray` value, call `float()` on it